### ISSUE TITLE
Get-PCCustomer requires  -StartsWith when parameter -ResultSize is provided

### ISSUE THREAD

**Issue by slavizh** [Microsoft/PartnerCenterPowerShellModule#72]
# Steps to reproduce

> What steps can reproduce the defect?
> Please share the setup, commandline for vstest.console, sample project, target
> framework etc.

```powershell
Get-PCCustomer -ResultSize 300
mdlet Get-PCCustomer at command pipeline position 1
Supply values for the following parameters:
StartsWith:
```
## Expected behavior

> Share the expected output

Get-PCCustomer with -ResultSize should not require the use of -StartsWith parameter

## Actual behavior

> What is the behavior observed?

Get-PCCustomer with -ResultSize requires the use of -StartsWith parameter as well

## Diagnostic logs

> Please share test platform diagnostics logs.  
> The logs may contain test assembly paths, kindly review and mask those before sharing.

No need. Issue is easy reproducible.

## Environment

> Please share additional details about your environment.
> Version

```powershell
PS D:\> Get-Module PartnerCenterModule

ModuleType Version    Name                                ExportedCommands
---------- -------    ----                                ----------------
Script     0.10.0.0   PartnerCenterModule                 {Add-PCAuthentication, Add-PCCustomerRoleMember, Get-PCAddressRulesByMarket, Get-PCAuditRecord...}
PS D:\> $PSVersionTable.PSVersion

Major  Minor  Build  Revision
-----  -----  -----  --------
5      1      17134  137
```

**Comment by joelst** [comment]
A fix for this has been checked in. Please test and reopen the issue, if it has not been resolved to your satisfaction.

**Comment by slavizh** [comment]
@joelst Can you specify in which version these are fixed or they will be fixed. It is hard to follow the changes of this module without proper releases like AzureRM.
